benchmarks/benchmark_base.py

```diff
@@ -350,13 +351,66 @@ def generate_reference(self):
             return option
 
     @property
-    def tardis_ref_data(self):
-        # TODO: This function is not working in the benchmarks.
-        if self.generate_reference:
-            mode = "w"
-        else:
-            mode = "r"
-        with pd.HDFStore(
-            f"{self.tardis_ref_path}/unit_test_data.h5", mode=mode
-        ) as store:
-            yield store
+    def verysimple_radfield_mc_estimators(self):
+        plasma = self.nb_simulation_verysimple.plasma
+        return radfield_mc_estimators.initialize_estimator_statistics(
+            plasma.tau_sobolevs.shape, plasma.gamma.shape
+        )
+
+    @property
+    def montecarlo_configuration(self):
+        return montecarlo_configuration.MonteCarloConfiguration()
+
+    @property
+    def rpacket_tracker(self): 
+        return RPacketTracker(0)
+
+    @property
+    def transport_state(self):
+        return self.nb_simulation_verysimple.transport.transport_state
+
+    @property
+    def simulation_rpacket_tracking_enabled(self):
+        config_verysimple = self.config_verysimple
```

Comment:
(Can be future work) Setting up a separate config property or a yml file is cleaner.